elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/AggregationStoreProperties.java

```diff
@@ -18,4 +20,8 @@
      */
     private boolean enabled = false;
 
+    /**
+     * {@link SQLDialect} type for default DataSource Object.
+     */
+    private String defaultDialect = "MYSQL";
```

Comment:
We should set to one of the supported values.

Reply:
changed to Hive.